jdaviz/configs/cubeviz/plugins/moment_maps/tests/test_moment_maps.py

```diff
@@ -116,20 +116,16 @@ def test_write_momentmap(cubeviz_helper, spectrum1d_cube):
     assert plugin._obj.overwrite_warn is False
 
     # Attempt to write the moment map to the same file we created earlier.
-    plugin._obj.filename = str(test_file)
+    plugin._obj.filename = test_file_str
     plugin._obj.vue_save_as_fits()
 
     # We should get an overwrite warning
     assert plugin._obj.overwrite_warn is True
     # and shouldn't have written anything (the file should be in tact)
-    with open(str(test_file.resolve()), 'r') as f:
-        assert existing_sentinel_text in f.read()
+    assert test_file.read_text() == existing_sentinel_text
 
     # Force overwrite the existing file
     plugin._obj.vue_overwrite_fits()
 
     # Read back in the file and check that it is equal to the one we calculated
-    assert_allclose(moment.data, CCDData.read(str(test_file)).data)
-
-    # Cleanup
-    test_file.unlink()
+    assert_allclose(moment.data, fits.getdata(test_file_str))
```

Comment:
FITS has less overhead than CCDData, but if you actually want to test `CCDData.read` here indirectly, I can change this back.

Reply:
Hmm... it wasn't my intention to test `CCDData.read` but I suppose if the output format is a `CCDData` object, then maybe it would be a good thing to test for indirectly? I'm on the fence about this one... I'll leave it up to you

Reply:
If it wasn't your intention, then let FITS to this job. CCDData is already tested in astropy anyway. Thanks!